tests/testthat/test-as-data-frame.R

```diff
@@ -122,6 +122,140 @@ with_mock_crunch({
         expect_identical(.crunchPageSize(ds$starttime), 100000L)
         expect_identical(.crunchPageSize(2016 - ds$birthyr), 50000L)
     })
+    
+    test_that("can manipulate the order of a crunchDataFrame", {
+        ds_df <- as.data.frame(ds)
+        gndr <- ds_df$v1
+        new_order <- c(4,3,2,1)
```

Comment:
Perhaps shuffle this so it's not just reversed, just to make it clear that it takes this order, whatever it is.